data/json/furniture_and_terrain/furniture-alien.json

```diff
@@ -692,5 +692,56 @@
     "move_cost_mod": -1,
     "required_str": -1,
     "flags": [ "TRANSPARENT" ]
+  },
+  {
+    "type": "furniture",
+    "id": "f_spiked_officer",
```

Comment:
I would possibly rename this to `f_impaled_officer`.
"Spiked" sounds like it is now some sort-of crystal cactus with spikes protuding from the body.